File: tests/fixtures/multilang/sample.php
<?php
require_once "helper.php";

// Adds two ints.
function add($a, $b) {
  if ($a > 0) {
    return helper::scale($a, $b);
  }
  return $a + $b;
}

class Widget {
  // Renders the widget.
  function render() {
    $this->add(1, 2);
    add(3, 4);
  }
}

File: tests/fixtures/multilang/Sample.cs
using System;

class Widget {
  /// Adds two ints.
  int Add(int a, int b) {
    if (a > 0) {
      return Helper.Scale(a, b);
    }
    return a + b;
  }

  // Renders the widget.
  void Render() {
    this.Add(1, 2);
  }
}
